### PR TITLE
Fix/GitHub actions

### PR DESCRIPTION
**Pull request by Leon-Bichmann**
This PR tries to fix current encountered github_actions issues #5223

### CODE REVIEW

**Comment by Leon-Bichmann** [comment]
Ok, so resetting the CTEST_BUILD_FLAG option makes the CI builds run on unix and osx in about 55min. Not sure how to include this option on windows though and I think that is why we tried to remove it.

About the THIRDPARTY tests : Im not sure how to include them.

Neither how to change the CMake version.

**Comment by jpfeuffer** [comment]
MSBuild for Windows has just different options. I would try "/p:CL_MPCount=NUMBER".

Clone the THIRDPARTY submodule, flatten the directory structure and add them to the path:
https://github.com/OpenMS/OpenMS/blob/develop/tools/travis/lnx-cibuild.before.sh#L63
https://github.com/OpenMS/OpenMS/blob/develop/tools/travis/lnx-cibuild.sh#L63

You have to google for how to install different CMake version on GHactions Windows. You probably have to use either chocolatey or an action from the marketplace.

**Comment by Leon-Bichmann** [comment]
Nice that the builds are working again.

**Comment by oliveralka** [comment]
I don't get why the win build is so much slower than the others.

File: .github/workflows/openms-ci.yml [hunk 1]
```diff
@@ -94,8 +102,10 @@ jobs:
           WITH_GUI: "ON"
           ADDRESS_SANITIZER: "Off"
           BUILD_TYPE: "Release"
-          OPENMP: "On"
+          OPENMP: "Off"
```
Comment (Leon-Bichmann) [inline .github/workflows/openms-ci.yml:105]: Not sure either - I thought its because of this. @jpfeuffer why did you disable it?

**Comment by Leon-Bichmann** [comment]
Hey,
What is the status on this PR, anything still missing? Do we leave the windows build as it is?

File: .github/workflows/openms-ci.yml (hunk 1 above)
Comment (jpfeuffer) [reply .github/workflows/openms-ci.yml:105]: I hoped that it would reduce compile times

**Comment by jpfeuffer** [comment]
you can have a look at the associated issue. I think the only thing that would help on windows is a compiler cache.

**Pull request merged**